src/App.tsx

```diff
@@ -1,24 +1,172 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+import {
+  addTodos,
+  deleteTodos,
+  editTodos,
+  getTodos,
+} from './api/todos';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { Notification } from './components/Notification';
+import { TodoItem } from './components/TodoItem';
+import { Errors } from './types/Errors';
+import { Todo } from './types/Todo';
+import { TodoStatus } from './types/TodoStatus';
 import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+const USER_ID = 6398;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedStatus, setSelectedStatus] = useState(TodoStatus.All);
+  const [errorType, setErrorType] = useState(Errors.None);
+  const [hasErrorNotification, setHasErrorNotification] = useState(false);
+  const [isUpdatingIds, setIsUpdatingIds] = useState<number[]>([]);
```

Comment:
probably some boolean variables can start with `is`
let's think more about variable name for this ids